dashboard/views_test.py

```diff
@@ -238,13 +239,33 @@ def test_enrollment_fails(self, mock_refresh, mock_edx_enr):  # pylint: disable=
         """
         Test error when backend raises an exception
         """
-        mock_edx_enr.side_effect = HTTPError()
+        error = HTTPError()
+        error.response = MagicMock()
+        error.response.status_code = status.HTTP_500_INTERNAL_SERVER_ERROR
+        mock_edx_enr.side_effect = error
         resp = self.client.post(self.url, {'course_id': self.course_id}, format='json')
         assert resp.status_code == status.HTTP_500_INTERNAL_SERVER_ERROR
         assert mock_edx_enr.call_count == 1
         # assert just the second argument, since the first is `self`
         assert mock_edx_enr.call_args[0][1] == self.course_id
 
+    @patch('edx_api.enrollments.CourseEnrollments.create_audit_student_enrollment', autospec=True)
+    @patch('backends.utils.refresh_user_token', autospec=True)
+    def test_enrollment_fails_configuration(self, mock_refresh, mock_edx_enr):  # pylint: disable=unused-argument
```

Comment:
It looks like this is basically the same test as above. Can we combine the two tests?